cetmix_tower_server/migrations/14.0.0.4.19/post-migration.py

```diff
@@ -0,0 +1,21 @@
+import logging
+
+from odoo import SUPERUSER_ID, api
+
+_logger = logging.getLogger(__name__)
+
+
+def migrate(cr, version):
+    """
+    Set default value for `skip_host_key` field to `True`
+    for all existing servers.
+    """
+    _logger.info("Starting migration for `skip_host_key` field.")
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    servers = env["cx.tower.server"].search([])
```

Comment:
What if someone has already updated the code and put host keys? 
Now they will have host check disabled even if they have valid keys saved in their server settings.

Reply:
I agree, fixed